spark/v3.5/spark/src/jmh/java/org/apache/iceberg/spark/action/IcebergSortCompactionBenchmark.java

```diff
@@ -280,6 +280,7 @@ protected Configuration initHadoopConf() {
     return new Configuration();
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
I'd prefer to fix the underlying issue here rather than just suppressing this.

```
-    SparkSessionCatalog catalog;
+    SparkSessionCatalog<?> catalog;
     try {
       catalog =
-          (SparkSessionCatalog) Spark3Util.catalogAndIdentifier(spark(), "spark_catalog").catalog();
+          (SparkSessionCatalog<?>) Spark3Util.catalogAndIdentifier(spark(), "spark_catalog").catalog();
```